src/scripts/GameConstants.ts

```diff
@@ -280,7 +280,19 @@ namespace GameConstants {
         "Bruno" = 10,
         "Agatha" = 11,
         "Lance" = 12,
-        "Champion" = 13
+        "Champion" = 13,
+        "Zephyr" = 14,
```

Comment:
Wouldn't this be easier when we divide it by region? Like with routes?

Reply:
Yeah that could deal with a couple of issues (Bruno being in Kanto and Johto, each region having a champion)

Reply:
How would you keep it as an enum though? Maybe have a KantoBadges enum and a JohtoBadges enum, then something like
```
Badges = {
  0: KantoBadges,
  1: JohtoBadges
}
```